lldb/source/Commands/Options.td

```diff
@@ -1117,6 +1117,11 @@ let Command = "thread plan list" in {
     Desc<"Display thread plans for unreported threads">;
 }
 
+let Command = "thread select" in {
+  def thread_select_thread_id : Option<"thread-id", "t">, Group<2>,
+    Arg<"ThreadID">, Desc<"Provide a thread ID instead of a thread index.">;
```

Comment:
We want to make sure this type is a `lldb::tid_t` (which is a `uint64_t`). Does `Arg<"ThreadID">` take care of this for us? I want to make sure we aren't defaulting to a 32 bit integer where someone might not be able to specify the full thread ID without it getting trucated. Can we verify this is using a `uint64_t`?